lib/go/test/Makefile.am

```diff
@@ -85,6 +86,17 @@ gopath: $(THRIFT) $(THRIFTTEST) \
 	cp -r ./dontexportrwtest gopath/src
 	touch gopath
 
+gopathfuzz: $(THRIFT) fuzz/fuzz.go
+	cd fuzz && $(THRIFT) -r --gen go ../../../../tutorial/tutorial.thrift
+	cd fuzz && go mod init fuzz
+	cd fuzz/gen-go/shared && go mod init shared
+	cd fuzz/gen-go/tutorial && go mod init tutorial
+	cd fuzz && go mod edit -replace shared=./gen-go/shared
+	cd fuzz && go mod edit -replace tutorial=./gen-go/tutorial
+	cd fuzz && go mod edit -replace github.com/apache/thrift/lib/go/thrift=../../../../lib/go/thrift
+	go build fuzz
```

Comment:
this line should be `cd fuzz && go build` instead

(does make return to your current directory after every command? I see we did a lot of `cd fuzz` in the commends but never `cd ..`)

Reply:
Also if we make this a rule in the fuzz directory instead of the test directory, we won't need to do `cd fuzz` in every command.

Reply:
> does make return to your current directory after every command?

It seems so (I tried `pwd` and I saw I was not in `fuzz` subdirectory)

Reply:
> Also if we make this a rule in the fuzz directory instead of the test directory

Trying this